meeting/meeting_march_8_2022.md

```diff
@@ -0,0 +1,63 @@
+# March 8th (6 PM CST)/9th (9 AM JST) Meeting 2022
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+
+* **#Query(Chris)**
+
+* **#Query(Matt)**
+
+* **#Query(Tibo)**
+
+* **#Query(Jesse)**
+
+* **#Query(Ben)**
+
+* **#Query(Lloyd)**
+
+* **#Query(Aki)**
+
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* others?
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* Changing serialization breaks contract ids
+  * https://mailmanlists.org/pipermail/dlc-dev/2022-February/000116.html
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/188
+  * Any Updates?
+    * **#Query(Tibo, Chris)**
+* Mutual Close Message
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/170
+  * Any Updates?
+    * **#Query(Matt)**
```

Comment:
bitcoin-s has a PR, there are still a couple todos in the spec PR however before it can be merged

https://github.com/bitcoin-s/bitcoin-s/pull/4065

Reply:
The remaining todos on https://github.com/discreetlogcontracts/dlcspecs/pull/170 are blocked by https://github.com/discreetlogcontracts/dlcspecs/pull/163 being merged